client/my-sites/themes/theme-options.js

```diff
@@ -97,10 +98,17 @@ function getAllThemeOptions() {
 	const deleteTheme = {
 		label: translate( 'Delete' ),
 		action: confirmDelete,
-		hideForTheme: ( state, themeId, siteId, origin ) =>
-			! isJetpackSite( state, siteId ) ||
-			origin === 'wpcom' ||
-			isThemeActive( state, themeId, siteId ),
+		hideForTheme: ( state, themeId, siteId, origin ) => {
+			const recommendedThemes = getRecommendedThemes( state );
+			const isThemeRecommended = recommendedThemes.some( ( recTheme ) => recTheme.id === themeId );
+
+			return (
+				! isJetpackSite( state, siteId ) ||
+				origin === 'wpcom' ||
+				isThemeActive( state, themeId, siteId ) ||
+				isThemeRecommended
+			);
```

Comment:
What if the theme is installed and recommended at the same time? Then I should be able to delete it from the Jetpack site. But this check will prevent that.

Reply:
I think you're right. Another approach coming.